src/basho_bench_profiler.erl

```diff
@@ -0,0 +1,65 @@
+-module(basho_bench_profiler).
+
+-export([ maybe_start_profiler/1, maybe_terminate_profiler/1
```

Comment:
This spacing seems a bit goofy...should be fine all on one line w/o the leading space?